lib/ship.rb

```diff
@@ -1 +1,19 @@
 #Ship Class
+require 'pry'
+class Ship
+  attr_reader :name, :length, :health
+
+  def initialize(name, length)
+    @name = name
+    @length = length
```

Comment:
I believe we'll want to remove the @ on the @health = @length variable due to scope if I understand it correctly.

Reply:
Just tested this and removing the @ and replacing it with just "length" seems to work! Scope is not my strong suit, thanks for catching that. Will push changes soon.